http/fetch/go.mod

```diff
@@ -14,6 +14,7 @@ replace github.com/opencontainers/go-digest => github.com/opencontainers/go-dige
 require (
 	github.com/fluxcd/pkg/tar v0.4.0
 	github.com/fluxcd/pkg/testserver v0.5.0
+	github.com/go-logr/logr v1.2.4
```

Comment:
Please bump it to v1.3.0 like we have in runtime https://github.com/fluxcd/pkg/blob/0603b64045a617a2599566baed7f868c55cf561c/runtime/go.mod#L18